portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@WebServlet("/delete-data")
+public class DeleteDataServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    List<Entity> allCommentEntities = DataServlet.getCommentEntities(datastore);
```

Comment:
Ok this is the question you had during standup, so will address it here.

First off, for THIS review, and in accordance with the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)), I'll let you get away with it. You can leave this as-is and not worry about it for this PR.

Now... if this were a more involved project, here's what I would have you do (again, you do not need to do this):

* Create a separate class that knows how to talk to your datastore and return entities.
* Have your two servlets depend on that class, rather than directly on the data store.

So you would have a class called `CommentsDataAccessor` (or some name like that), which would have a `DataStoreService` as a member, and it would have `getComments` and `deleteAllComments` methods. It would have other stuff too.

There are a lot of advantages to having something like this. Mainly, it creates much stronger separation of concerns. The Servlet should only care about basic request validation and communicating between with the client.

The CommentsDataAccessor should only care about what it means for there to be "comments" and how to access them.